content/en/cosign/signing_with_blobs.md

```diff
@@ -95,3 +48,51 @@ tlog entry created with index: 965333
 Signature wrote in the file sig
 Certificate wrote in the file cert.pem
 ```
+
+
+## Signing with a key
+
+You may specify your own keys for signing.  You will need the password for the private key to sign:
+
+```shell
+$ cosign sign-blob --key cosign.key README.md
+Using payload from: README.md
+Enter password for private key:
+MEQCIAU4wPBpl/U5Vtdx/eJFgR0nICiiNCgyWPWarupH0onwAiAv5ycIKgztxHNVG7bzUjqHuvK2gsc4MWxwDgtDh0JINw==
+```
+
+This supports all the same flags and features as `cosign sign`, including KMS support, hardware tokens, and keyless signatures.
+
+## Blobs in OCI Registries
+
+You can upload blobs to an OCI registry (similar to [ORAS](https://github.com/oras-project/oras)) where they can then be signed like any other image.
+
+You can publish an artifact with `cosign upload blob`:
+
+```shell
+$ echo "my first artifact" > artifact
+$ cosign upload blob -f artifact gcr.io/user/demo/artifact
+Uploading file from [artifact] to [gcr.io/user/demo/artifact:latest] with media type [text/plain; charset=utf-8]
+File is available directly at [us.gcr.io/v2/user/demo/readme/blobs/sha256:b57400c0ad852a7c2f6f7da4a1f94547692c61f3e921a49ba3a41805ae8e1e99]
+us.gcr.io/user/demo/readme@sha256:4aa3054270f7a70b4528f2064ee90961788e1e1518703592ae4463de3b889dec
+```
+
+Your users can download it from the "direct" URL with standard tools like curl or wget:
+
+```shell
+$ curl -L gcr.io/v2/user/demo/artifact/blobs/sha256:97f16c28f6478f3c02d7fff4c7f3c2a30041b72eb6852ca85b919fd85534ed4b > artifact
+```
+
+The digest is baked right into the URL, so users can check that as well:
```

Comment:
Would it be correct to say "included in the URL" so that this is accessible to non-native English speakers?